redhat-developer/developers.redhat.com
# DEVELOPER-4726 Add publish date field

Adding a new field to cheat-sheets and training classes. The field won't display unless there's content in the field.

### JIRA Issue Link
* https://issues.jboss.org/browse/DEVELOPER-4726

### Verification Process
* Visit a cheat-sheet, and video resource (no idea where to show the date currently in a Training Resource)
* Verify the publish date is not displayed (unless the field is filled in)
* Modify a cheat-sheet and video resource to add the publish date
* Visit the modified node
* Verify the publish date is displayed
